touchin-custom-view/Code/View/view_controller.swift.liquid

```diff
@@ -0,0 +1,56 @@
+import UIKit
+import LeadKit
+import RxSwift
+import RxCocoa
+
+final class {{ module_info.name }}ViewController: BaseCustomViewController<{{ module_info.name }}ViewModel, {{ module_info.name }}View>, {{ module_info.name }}Module, DisposeBagHolder {
+
+    // MARK: - Overrides
+
+    // MARK: - Views
+
+    // MARK: - Properties
+
+    let disposeBag = DisposeBag()
+
+    // MARK: - View Life Cycle
+
+    // MARK: - ConfigurableController
+
+    override func bindViews() {
+        super.bindViews()
+
+        <#code#>
+    }
+
+    override func configureAppearance() {
+        super.configureAppearance()
+
+        <#code#>
+    }
+
+    override func localize() {
+        super.localize()
+
+        <#code#>
+    }
+
+    override func configureBarButtons() {
+        super.configureBarButtons()
+
+        <#code#>
+    }
+
+    // MARK: - Actions
+
+}
+
+// MARK: - Public
+
+extension {{ module_info.name }}ViewController {
+}
```

Comment:
ViewController is hidden behind presentable

Reply:
I don't get it..?

Reply:
you can't call public method of VC coz your VC is behind Presentable protocol

Reply:
this is actually internal... I'd better rename it